test/kubemark/resources/hollow-node_template.json

```diff
@@ -145,27 +157,48 @@
 				},
 				{
 					"name": "hollow-node-problem-detector",
-					"image": "gcr.io/google_containers/node-problem-detector:v0.2",
+					"image": "gcr.io/google_containers/node-problem-detector:v0.3",
```

Comment:
@wojtek-t @shyamjvs The entrypoint in the image will be overwritten by the command defined in pod yaml. (See the code in dockertools https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockertools/docker_manager.go#L843-L848, and the explanation in api https://github.com/kubernetes/kubernetes/blob/55bee3ad21f025b1416a4e1f10de753f484b66d3/pkg/api/types.go#L1284)